### PR TITLE
Added docusign get documents and update database. Save doc to gdrive

### PR DESCRIPTION
**Pull request by ValdarRudman**

### CODE REVIEW

**Comment by ValdarRudman** [comment]
Fixed up and added code. Possible few more commits for this.

File: credentials.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{"installed":{"client_id":"181032288929-dsna30mqeooco6fluvdennrtansnq268.apps.googleusercontent.com",
+  "project_id":"quickstart-1601290410416","auth_uri":"https://accounts.google.com/o/oauth2/auth",
+  "token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs",
```
Comment (SeanCondon) [inline credentials.json:3]: could some of these values be moved to the ENV file?

File: src/tsweekly/docusign/send-email-sign.ts [hunk 2]
```diff
@@ -70,50 +63,43 @@ signingViaEmail.worker = async (args) => {
...
 
-  // add the documents
-  const doc1 = new docusign.Document();
+  // add 1 day reminder
+  const notification = new docusign.Notification();
+  notification.useAccountDefaults = 'false';
+
+  const reminders = new docusign.Reminders();
+  reminders.reminderEnabled = 'true';
+  reminders.reminderDelay = '1';
```
Comment (SeanCondon) [inline src/tsweekly/docusign/send-email-sign.ts:75]: nice

File: token.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (SeanCondon) [inline token.json:1]: can some of the strings here be moved to the ENV file?

**Review by SeanCondon — APPROVED** [review]
A couple of minot comments. If it's not possible to move the variables to ENV, then no problem

File: token.json (hunk 3 above)
Comment (ValdarRudman) [reply token.json:1]: No these can't. This is created at run time (though we should be able to move one in). This is where google looks for the token when authenticating

File: credentials.json (hunk 1 above)
Comment (ValdarRudman) [reply credentials.json:3]: Possible, however this seems to be googles standard. Some information is extracted, I am just not sure of all the locations this file is used

**Pull request merged**